crates/red_knot_test/src/config.rs

```diff
@@ -3,26 +3,41 @@
 //! following limited structure:
 //!
 //! ```toml
+//! log = true # or log = "red_knot=WARN"
 //! [environment]
 //! python-version = "3.10"
 //! ```
```

Comment:
should we also add this to the example at https://github.com/astral-sh/ruff/tree/main/crates/red_knot_test#configuration ?

Reply:
I added a link to the `MarkdownTestConfig`. I don't think we need to list all options